writer.go

```diff
@@ -1207,7 +1207,9 @@ func (c *writerColumn) flushFilterPages() error {
 		// be reused after resetting which would have reset the length of
 		// the filter to 0.
 		c.resizeBloomFilter(int64(dict.Len()))
-		return c.writePageToFilter(dict.Page())
+		dictPage := dict.Page()
+		defer Release(dictPage)
```

Comment:
Not sure whether the additions to `writer.go` here are necessary, but made them anyway so that we can discuss. Will remove if not necessary.

Reply:
I believe this is needed 👍